.gitignore

```diff
@@ -25,3 +25,6 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+# ignore debug config
+.vscode
```

Comment:
`.vscode` folder shouldn't be ignored since it's often great to share some workspace configs